source/_posts/2019-09-10-hexo-filter-nofollow-joined-hexo-official-plugin.md

```diff
@@ -0,0 +1,25 @@
+---
+title: hexo-filter-nofollow joined hexo official plugins
+---
+
+We released a new hexo official plugin: [hexo-filter-nofollow]
+
+[hexo-filter-nofollow] will add `rel="external nofollow noreferrer"` to all external links automatically for security, privacy and SEO. [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types).
+
+You can install this plugin using the command below:
+
+
+``` bash
+$ npm i hexo-filter-nofollow --save
+```
+
+And add configuration to your hexo's `_config.yml`:
```

Comment:
and enable it through your hexo's \`_config.yml\`:

Reply:
Updated